packages/dataprovider/README.md

```diff
@@ -546,3 +546,26 @@ const dataProvider = useDataProvider({
     queryDialect: "typegraphql" // 👈
 })
 ```
+
+### override mutation operation names due to prisma versions breaking changes
+
+Now you can override mutation calls with your prefixes
+
+```ts
+import { Options } from "@ra-data-prisma/dataprovider";
+import { makePrefixedFullName } from "@ra-data-prisma/dataprovider/lib/utils/makePrefixedFullName";
```

Comment:
its probably better to export it from top-level or some barrel-file, othrwise we expose internal folder structure that might change in the future

Reply:
Ahh yea you are right!